docker-compose.yml

```
@@ -20,6 +20,7 @@ services:
       - DJANGO_SECRET_KEY=not-secret-in-dev
       - ELASTICSEARCH_URL=elasticsearch:9200
       - GOOGLE_MAPS_GEOCODING_API_KEY=${GOOGLE_MAPS_GEOCODING_API_KEY}
+      - DJANGO_DEBUG_TOOLBAR=true
```

If you leave this blank, you can set it when needed using a `.env` file in the project directory, or a variable password to docker-compose.

```suggestion
      - DJANGO_DEBUG_TOOLBAR
```